.scalafmt.conf

```
@@ -12,3 +12,18 @@ project.excludeFilters = [
   scalafix-tests/shared/src/main/scala/test/PrettyTest.scala
   scalafix-sbt/src/main/scala/scalafix/internal/sbt/ScalafixRuleNames.scala
 ]
+
+fileOverride {
+  "glob:**/scalafix-tests/integration/src/test/scala-3/**" {
+     runner.dialect = scala3
+  }
+  "glob:**/scalafix-reflect/src/main/scala-3/**" {
+     runner.dialect = scala3
+  }
+  "glob:**/scalafix-core/src/main/scala-3/**" {
+     runner.dialect = scala3
+  }
+  "glob:**/scalafix-tests/shared/src/main/scala-3/**" {
+     runner.dialect = scala3
+  }
```

can the glob match arbitrary depth?
```suggestion
  "glob:**/scala-3/**" {
     runner.dialect = scala3
  }
```

---

I believe so yes, that's how we have it in Metals, which is how I copied these over.

---

could we take in my suggestion then to avoid hardcoding module paths?

---

Hey, yup, sorry for the delay.